src/main/java/io/debezium/connector/vitess/VitessPartition.java

```diff
@@ -10,20 +10,31 @@
 import java.util.Objects;
 import java.util.Set;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import io.debezium.pipeline.spi.Partition;
 import io.debezium.util.Collect;
 
 public class VitessPartition implements Partition {
-    private static final String SERVER_PARTITION_KEY = "server";
+    private static final Logger LOGGER = LoggerFactory.getLogger(VitessPartition.class);
```

Comment:
nit: I think logger is not used in this class?

Reply:
Nice catch, fixed in the new commit.